packages/outputarea/src/widget.ts

```diff
@@ -176,6 +176,13 @@ export class OutputArea extends Widget {
    */
   readonly outputLengthChanged = new Signal<this, number>(this);
 
+  /**
+   * A static signal that emits when a page payload is requested.
+   */
+  static pageRequested = new Signal<typeof OutputArea, OutputArea.IPageRequest>(
```

Comment:
That static signal doesn't look right (both the `static` and the fast that it's a `Signal` and not `ISignal`).

Normally the pattern is to expose an `ISignal` so consumers of the signal can't emit the signal themselves.